pepys_timeline/static/js/src/pepys.js

```diff
@@ -1,10 +1,15 @@
+/* eslint-env browser */
+
 moment.locale("en");
 
 const DATE_FORMATS = {
   visavail: "YYYY-MM-DD HH:mm:ss",
   metadata: "YYYY-MM-DD",
   picker: "DD/MM/YYYY",
 }
+const DEFAULT_MESSAGE_OF_THE_DAY = 'Manager of the day: Daisy Duke';
```

Comment:
If we don't have a database connection, or can't get the field from the database, then let's have the default message say something valid `Message of the day: [PENDING]`

Reply:
Noted!